README.md

```diff
@@ -163,6 +166,190 @@ Then the page should be accessible checking only: document-title, label
 Then the page should be accessible according to: best-practice and checking: aria-roles, definition-list
 ```
 
+# RSpec
+
+## Configuration
+
+1. Require rspec matchers: in `spec/spec_helper.rb`, `spec/support/env.rb` or similar.
+
+    ``` ruby
+    require 'axe/rspec'
+    ```
+
+2. Configure Browser/Webdriver
```

Comment:
The configuration shouldn't be necessary for rspec, since the page/driver is passed explicitly to the matcher ala: `expect(page)`.